.github/workflows/tests-ui.yml

```diff
@@ -1,4 +1,4 @@
-name: UI automation tests
+name: UI automation and a11y Lighthouse tests
```

Comment:
This is a little long, can we come up with a simpler name that captures it all?

(See the very bottom): 
![image](https://user-images.githubusercontent.com/1783439/142514784-56032a10-6a4b-477e-b644-5c2394c0d00c.png)

Reply:
👍